profiler/options.go

```diff
@@ -63,6 +63,9 @@ const (
 	defaultAgentPort = "8126"
 )
 
+// Path to the default trace agent Unix domain socket, replaced by tests
+var defaultSocketAPM = internal.DefaultTraceAgentUDSPath
```

Comment:
This seems to be dead code? I don't see anything referencing it. Tests still pass if I remove it.

Reply:
Yeah, it's dead code, not needed after 9b525fc. Thanks for catching it.